job_bundles/README.md

```diff
@@ -0,0 +1,308 @@
+and then written to a temporary file within the Session Working Directory 
+(see: [Sessions](https://github.com/OpenJobDescription/openjd-specifications/wiki/How-Jobs-Are-Run#sessions)) on disk.
+Note that the `BlenderSceneFile` and `OutputDir` Job Parameters are both defined with `type: PATH`, so `{{Param.BlenderSceneFile}}` and
+`{{Param.OutputDir}}` will resolve to the [path-mapped location](https://github.com/OpenJobDescription/openjd-specifications/wiki/How-Jobs-Are-Run#path-mapping)
+on the Worker where the file and directory are located.
+After the embedded file has been written
+to disk, the Task's `onRun` command is resolved to `bash <location-of-embeded-file>` and then run.
+
+Please see the Open Job Description specification for additional details on [how Jobs are run](https://github.com/OpenJobDescription/openjd-specifications/wiki/How-Jobs-Are-Run).
+
+Additional samples of Job Templates can be found in all of the Job Bundles in this repository as well as in the 
+[samples provided](https://github.com/OpenJobDescription/openjd-specifications/tree/mainline/samples)
+by Open Job Description. Please see https://github.com/OpenJobDescription/openjd-specifications/wiki
+for more information on Job Templates.
+
+### Elements - Parameter Values
+
+The `parameter_values.yaml`/`parameter_values.json` file in a Job Bundle gives you a place to "bake"
+the values of some of the Job Parameters, and/or [deadline:CreateJob API](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_CreateJob.html)
+request arguments into the Job Bundle so that the values do not have to be given when submitting a Job using the
+Job Bundle. The Job Bundle submission UI that is created will still allow values for these parameters to be modified.
+
+The format of the file, in YAML, is:
+
+```yaml
+parameterValues:
+- name: <string>
+  value: <integer>, <float>, or <string>
+- name: <string>
+  value: <integer, <float>, or <string>
+... repeating as necessary
+```
+
+Each element of the `parameterValues` list in the file must be one of the following: 
+
+1. A Job Parameter defined in the Job Bundle's Job Template; 
+2. A Job Parameter defined in a Queue Environment on the Queue that you are submitting the Job to;
+3. A special parameter that is passed to the [deadline:CreateJob API](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_CreateJob.html)
+   when creating a Job with the Job Bundle:
+    * `deadline:priority` - The value must be an integer, and is passed as the `priority` request parameter to the API.
+    * `deadline:targetTaskRunStatus` - Value must be a string, and is passed as the `targetTaskRunStatus` request parameter to the API.
+    * `deadline:maxFailedTasksCount` - Value must be an integer, and is passed as the `maxFailedTasksCount` request parameter to the API.
+    * `deadline:maxRetriesPerTask` - Value must be an integer, and is passed as the `maxRetriesPerTask` request parameter to the API.
```

Comment:
is `deadline:storageProfileId` supported?

Reply:
[Not at this time.](https://github.com/aws-deadline/deadline-cloud/blob/c6334e37ca13f676ad8905a30f54c78d4cc081d5/src/deadline/client/ui/widgets/shared_job_settings_tab.py#L287-L304)